Library/Formula/rabbitmq.rb

```diff
@@ -53,6 +50,12 @@ def caveats; <<-EOS.undent
     EOS
   end
 
+  test do
+    system sbin/"rabbitmq-server", "-detached"
+    system sbin/"rabbitmqctl", "status"
+    system sbin/"rabbitmqctl", "stop"
+  end
```

Comment:
The old and new versions behave the same: the both fail without [the line](https://github.com/MerelyAPseudonym/homebrew/blob/de5ebe3f037701187afbdbbdb21badebc6d24795/Library/Formula/rabbitmq.rb#L56) that sets the `RABBITMQ_MNESIA_BASE` environment variable to something within the sandbox.